deployment/maintain/rebase-helper.md

```diff
@@ -8,10 +8,10 @@ order: 2
 Rebase-helper is a tool which helps package maintainers with updating package to the latest upstream version.
 It automates a lot of manual tasks the package maintainer usually does, when a new upstream version of a package is released.
 
-## How to install rebase-helper
+## Installing rebase-helper
```

Comment:
I'd preffer `Install` instead of `Installing`.

Reply:
Fair enough.